src/subscribables/dependentObservable.js

```diff
@@ -57,6 +57,34 @@ ko.computed = ko.dependentObservable = function (evaluatorFunctionOrOptions, eva
         isSleeping = false;
     }
 
+    function subscribeToDependency(target) {
```

Comment:
I had originally written this with static functions but then realized I needed to access `target` in one of them. I'll take another look at it, though, to see if there's another way. Also this will only run once for each dependency because when a computed is re-evaluated, it saves and reuses previous subscriptions.